contact.php

```diff
@@ -2,7 +2,7 @@
 session_start();
 ?>
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
There have been a few changes due to the html validator, this is one of them. The others are alt tags on images, id tags on the inputs, and removing the type attributes from the script and textarea tags